### DISCUSSION TITLE
绝区零一卡一卡的是为什么啊！求救！！！

### DISCUSSION THREAD

**Discussion by SW-GFrog** [share121/inter-knot#5149] [委托]
每次打绝区零的时候换地图，进地图，甚至进养成的时候都会直接卡住，明明电脑配置挺高的说，虚拟内存24G应该也够，画质调低也没用，是我网络问题还是绝区零优化啊，求大佬解答一下，新电脑配了才半年呢，4070ti

**Comment by meichuanlasi** [comment]
点了，我也出现了这样的问题

**Reply by SW-GFrog** [reply]
好像知道原因了（大概？），我的硬盘读写速度太慢了，读出来的速度不够，就会卡住，换了个盘，还没试过，下午开学

**Reply by faatong** [reply]
游戏建议放固态，过图换场景黑屏卡住的话就是读写太慢
